cocos/scripting/lua-bindings/script/cocos2d/Cocos2dConstants.lua

```diff
@@ -182,6 +182,7 @@ cc.PLATFORM_OS_MAC     = 2
 cc.PLATFORM_OS_ANDROID = 3
 cc.PLATFORM_OS_IPHONE  = 4
 cc.PLATFORM_OS_IPAD    = 5
+cc.PLATFORM_OS_OHOS     = 6
```

Comment:
The value is conflict with `cc.PLATFORM_OS_BLACKBERRY`. It is better to use another number.

Reply:
modified